ui/src/components/WMErrorsTable/index.jsx

```diff
@@ -0,0 +1,249 @@
+          map(({ node: { errorId } }) => errorId)
+        )(errorsConnection.edges);
+
+        return `${ids.join('-')}-${sortBy}-${sortDirection}-${searchTerm}`;
+      },
+    }
+  );
+
+  handleHeaderClick = sortBy => {
+    const toggled = this.state.sortDirection === 'desc' ? 'asc' : 'desc';
+    const sortDirection = this.state.sortBy === sortBy ? toggled : 'desc';
+
+    this.setState({ sortBy, sortDirection });
+  };
+
+  handleDrawerClose = () => {
+    this.setState({
+      drawerError: null,
+    });
+  };
+
+  handleDrawerOpen = ({ currentTarget: { name } }) => {
+    const { errorsConnection } = this.props;
+    const drawerError = errorsConnection.edges.find(
+      ({ node }) => node.errorId === name
+    ).node;
+
+    this.setState({
+      drawerError,
+    });
+  };
+
+  renderTableRow = error => {
+    const { classes } = this.props;
+    const { errorId, title, description, reported } = error.node;
+    const iconSize = 16;
+
+    return (
+      <TableRow key={errorId}>
+        <TableCell>
+          <IconButton
+            className={classes.infoButton}
+            name={errorId}
+            onClick={this.handleDrawerOpen}>
+            <InformationVariantIcon size={iconSize} />
+          </IconButton>
+          <TableCellItem>
+            <ListItemText disableTypography primary={title} />
+          </TableCellItem>
+        </TableCell>
+        <TableCell>
+          <Typography className={classes.errorDescription}>
+            {description}
+          </Typography>
+        </TableCell>
+
+        <CopyToClipboard title={`${reported} (Copy)`} text={reported}>
+          <TableCell>
+            <TableCellItem button>
+              <ListItemText
+                disableTypography
+                primary={
+                  <Typography>
+                    <DateDistance from={reported} />
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellItem>
+          </TableCell>
+        </CopyToClipboard>
+      </TableRow>
+    );
+  };
+
+  render() {
+    const { classes, errorsConnection, searchTerm, onPageChange } = this.props;
+    const { sortBy, sortDirection, drawerError } = this.state;
+    const sortedErrors = this.sortErrors(
+      errorsConnection,
+      sortBy,
+      sortDirection,
+      searchTerm
+    );
+
+    return (
+      <Fragment>
+        <ConnectionDataTable
+          connection={sortedErrors}
+          pageSize={VIEW_WORKER_POOL_ERRORS_PAGE_SIZE}
+          sortByHeader={sortBy}
+          sortDirection={sortDirection}
+          onHeaderClick={this.handleHeaderClick}
+          renderRow={this.renderTableRow}
+          headers={['Title', 'Description', 'Reported']}
+          padding="dense"
+          onPageChange={onPageChange}
+        />
+        <Drawer
+          anchor="right"
+          open={Boolean(drawerError)}
+          onClose={this.handleDrawerClose}>
```

Comment:
The drawer is hard to close on mobile:

![Screen Shot 2019-09-27 at 10 15 54 AM](https://user-images.githubusercontent.com/3766511/65776281-cf49f380-e10f-11e9-9616-367faf502f3a.png)

We can do something similar to the drawer in the pulse-messages view

https://github.com/taskcluster/taskcluster/blob/816c24710f85ed8da561b1647f6246a32deb64e1/ui/src/views/PulseMessages/index.jsx#L65-L69

![Screen Shot 2019-09-27 at 10 16 26 AM](https://user-images.githubusercontent.com/3766511/65776325-e2f55a00-e10f-11e9-99b6-900a66f6a494.png)